acceptance-tests/actors/azure.go

```diff
@@ -51,19 +51,6 @@ func (z azureLBHelper) getLoadBalancer(resourceGroupName, loadBalancerName strin
 	return true, nil
 }
 
-func (z azureLBHelper) getApplicationGateway(resourceGroupName, applicationGatewayName string) (bool, error) {
-	_, err := z.applicationGatewaysClient.Get(fmt.Sprintf("%s-bosh", resourceGroupName), applicationGatewayName)
-	if err != nil {
-		if aerr, ok := err.(autorest.DetailedError); ok {
-			if aerr.StatusCode.(int) == 404 {
-				return false, nil
-			}
-		}
-		return false, err
-	}
-	return true, nil
-}
-
```

Comment:
this is an unused function correct?

Reply:
Yes - the removals were based on the IDE's static analysis.